pq-crypto/kyber_r3/kyber512r3_fips202x4_avx2.c

```diff
@@ -0,0 +1,670 @@
+#include <stddef.h>
```

Comment:
I think we discussed that it would be best not to change anything from Kyber's repo that is not necessary. So I think it'd be good if you copy the keccak4x folder as is from the repo.

Reply:
It is updated as it is implemented in Crystals Kyber repo.